app/models/foreman_chef/fact_importer.rb

```diff
@@ -4,6 +4,10 @@ def fact_name_class
       ForemanChef::FactName
     end
 
+    def self.authorized_smart_proxy_features
+      'Chef'
```

Comment:
Well for users that upgraded from 1.6 it would be 'Chef Proxy', for fresh 1.7 it does not matter (and does not work atm) but today I'd like to release fixing version that will need 'Chef'. So I think you can leave 'Chef' here.